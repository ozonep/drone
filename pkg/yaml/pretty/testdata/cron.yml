version: 1
name: nightly
kind: cron

spec:
  branch: master

  schedule: 1 * * * *

  deployment:

    target: production
