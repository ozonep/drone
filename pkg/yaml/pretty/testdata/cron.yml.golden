---
version: 1
kind: cron
name: nightly
spec:
  schedule: 1 * * * *
  branch: master
  deployment:
    target: production

...
